test/fail_compilation/checkimports3.d

```diff
@@ -0,0 +1,15 @@
+/*
+REQUIRED_ARGS: -transition=checkimports -de
+TEST_OUTPUT:
+---
+fail_compilation/checkimports3.d(14): Deprecation: local import search method found overloadset checkimports3.foo (3 overloads) instead of overloadset checkimports3.foo (2 overloads)
```

Comment:
Yes it's on by default unless you disable it with -transition=import, checkimports is to be used once for checking that transition.